### ISSUE TITLE
Investigate environments warning

### ISSUE THREAD

**Issue by johanandren** [akka/akka-persistence-couchbase#57]
When running the tests we get:

```
More than 1 Couchbase Environments found (2), this can have severe impact on performance and stability. Reuse environments
```

We need to figure out if this is something we do wrong or if it is harmless.

**Comment by johanandren** [comment]
Could it be because we start several journal instances/cluster connections in the same jvm in tests?

**Comment by ygree** [comment]
There are multiple points where we create CouchbaseSessions: CouchbaseJournal, CouchbaseSnapshotStore, CouchbaseReadJournal as well as in Lagom modules that are used for offset store and persistent read side.

**Comment by johanandren** [comment]
So I guess this will likely be solved by #77 then

**Comment by johanandren** [comment]
Closing this now that we are sharing a single client across the plugins.
